cyclops-ctrl/internal/cluster/k8sclient/client.go

```diff
@@ -246,6 +248,21 @@ func (k *KubernetesClient) GetStatefulSetsLogs(namespace, container, name string
 	return logs, nil
 }
 
+func (k *KubernetesClient) RestartDeployment(group, version, kind, name, namespace string) error {
+	deploy, err := k.clientset.AppsV1().Deployments(namespace).Get(context.Background(), name, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	if deploy.Spec.Template.ObjectMeta.Annotations == nil {
+		deploy.Spec.Template.ObjectMeta.Annotations = make(map[string]string)
+	}
+	deploy.Spec.Template.ObjectMeta.Annotations["kubectl.kubernetes.io/restartedAt"] = time.Now().Format(time.RFC3339)
+
+	_, err = k.clientset.AppsV1().Deployments(namespace).Update(context.Background(), deploy, metav1.UpdateOptions{})
+	return err
```

Comment:
Can you also implement this for statefulsets and daemonsets?

Reply:
Yes. I have implemented Statefulsets and Daemonsets in new commits. Please review and update me.